### PR TITLE
Need to improve LED documentation

### PR DESCRIPTION
**Pull request by adelgadoj3**
Added more structure to the file. Need to complete document.

### CODE REVIEW

File: micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
 https://github.com/oss-slu/Pi4Micronaut/edit/develop/micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc[Improve this doc]
 
+===== Overview
+This section contains an overview of the LED component, it's circuit, and other instructions.
+
+
+===== Components
+
+
+
```
Comment (ruthvikm) [inline micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc:13]: Include Assembly Instructions section

File: micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc [hunk 2]
```diff
@@ -22,3 +35,7 @@ include::../../../../../../micronautpi4j-utils/src/main/java/com/opensourcewiths
 ====== This controller turns the LED on and off
 
 TODO: Create an example LED Controller
```
Comment (ruthvikm) [inline micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc:37]: Get rid of this TODO

File: micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc [hunk 3]
```diff
@@ -22,3 +35,7 @@ include::../../../../../../micronautpi4j-utils/src/main/java/com/opensourcewiths
...
+
```
Comment (ruthvikm) [inline micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc:38]: Move the below sections below circuit diagram and above constructors

**Review by ruthvikm — CHANGES_REQUESTED** [review]

File: micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc [hunk 4]
```diff
@@ -1,8 +1,48 @@
...
+then the resistor to 3.3V, and connect the cathode (the short pin) of the LED to GPIO 17 of the RaspberryPi. In addition, there need to be two jumper cables connected to
+separate positions on the Breadboard and T-Extension Board. The first will go from the GPIO17 pin to the cathode. Then place the second wire from the 3.3V pin to the positve side
+of the Breadboard.
+
+
+===== Circuit Diagram
+
+image::https://github.com/oss-slu/Pi4Micronaut/tree/main/micronautpi4j-utils/src/docs/asciidoc/image49.png
+
+
```
Comment (ruthvikm) [inline micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc:29]: Add Testing section and mention all the curl commands we use for this led component

**Review by ruthvikm — CHANGES_REQUESTED** [review]

**Review by ruthvikm — CHANGES_REQUESTED** [review]
- you have altered Led Controller class by adding led2. It's not necessary because it falls under multi component structure. Revert it back to one ledhelper.
- delete the :omega: variable on top.
- use the syntax like image::../../img/LED_circuit.png[] for circuit diagram

File: components/src/main/java/com/opensourcewithslu/components/controllers/LEDController.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+@Controller("/led")
+public class LEDController {
+    private final LEDHelper ledHelper;
+
+
+    public LEDController(@Named("Led")DigitalOutput led){
+        this.ledHelper = new LEDHelper(led);
+    }
+
+    @Get("/led")
```
Comment (SwindleA) [inline components/src/main/java/com/opensourcewithslu/components/controllers/LEDController.java:19]: The documentation says that this should be LEDOn. Change this to be `@Get("/LEDOn")`

File: components/src/main/java/com/opensourcewithslu/components/controllers/LEDController.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    public void checkLed(){
```
Comment (SwindleA) [inline components/src/main/java/com/opensourcewithslu/components/controllers/LEDController.java:20]: Have this match what is in the @Get

**Review by SwindleA — CHANGES_REQUESTED** [review]
two small things then it should be good.

File: components/src/main/java/com/opensourcewithslu/components/controllers/LEDController.java (hunk 5 above)
Comment (ruthvikm) [reply components/src/main/java/com/opensourcewithslu/components/controllers/LEDController.java:19]: Also make use all of the LED helper methods in the controller. We have deviceOn, deviceOff and switchState.

File: micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc [hunk 7]
```diff
@@ -1,8 +1,63 @@
...
+* Connect the anode (long pin) to a 220Ω resistor on the positive side of the Breadboard.
+* Connect the other end of the resistor to 3.3V
+* Connect a jumper wire from GPIO 17 to the cathode of the LED.
+* Connect another jumper wire from 3.3V to the positive side of the Breadboard.
+* Two jumper wires should be connected to separate positions on the Breadboard and T-Extension Board.
+
+
+===== Circuit Diagram
+
+image::/../../../../../asciidoc/img/LED_circuit.png[]
```
Comment (ruthvikm) [inline micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/led.adoc:32]: Fix the image path

**Review by ruthvikm — CHANGES_REQUESTED** [review]

**Review by SwindleA — APPROVED** [review]
ok

**Review by ruthvikm — APPROVED** [review]
Documentation looks good. Need to make use all of the LED helper methods in the controller. We have deviceOn, deviceOff and switchState. We will work on it with a new issue.

**Pull request merged**